<?xml version="1.0"?>
<launch>
    <node pkg="mdr_push_action" type="push_action" name="push_server" output="screen">
        <param name="move_arm_server" value="move_arm_server" />
        <param name="move_base_server" value="move_base_server" />
        <param name="gripper_controller_pkg_name" value="mas_hsr_gripper_controller" />
        <param name="pregrasp_config_name" value="pregrasp" />
        <param name="grasping_dmp" value="$(find mdr_push_action)/config/trajectory_weights/weights_table_grasp.yaml" />
        <param name="dmp_tau" value="30" />
        <rosparam param="grasping_orientation">[0, 0, 0, 1]</rosparam>
        <param name="number_of_retries" value="0" />
    </node>
</launch>
